### ISSUE TITLE
Unpin dependencies in `teen_process`

### ISSUE THREAD

**Issue by notjosh** [appium/appium#17480]
## The problem

Hiya 👋

The dependencies for `teen_process@2` ([here](https://github.com/appium/node-teen_process)) are pinned to specific versions of, say, `@babel/runtime: "7.18.9"`.

I propose this be updated to allow version ranges (i.e. `"^7.0.0"`) as was the case in `teen_process@1` to reduce the likelihood of ending up with multiple versions of `@babel/runtime` in projects.

Either that, or more formally bring `teen_process` into the `@appium/` scope, so the relationship is clearer that versions should be kept in sync within projects.

Ideally Appium unpins Babel dependencies more generally, but that might be a bigger can of worms :)

## Environment

```
// current latest published versions:

appium@2.0.0-beta.44 depends on:
  @babel/runtime@7.19.0

appium-uiautomator2-driver@2.7.0 depends on:
  @babel/runtime@^7.0.0
  teen_process@^2.0.0

teen_process@2.0.1 depends on:
  @babel/runtime@7.18.9
```

## Details

While on one level this could be considered desired behaviour, in case you were doing (for example) careful AST traversal that relied on specific versions, I instead consider multiple concurrent `@babel/runtime` versions a smell that is best avoided.

Admittedly this example is unrelated, but to illustrate the point: we've had issues in past where `eslint` will parse with one version, but an `eslint`-plugin will parse with another version. The result is a non-sensical error that is resolved by simply using consistent Babel versions.

**Comment by mykola-mokhnach** [comment]
I totally support that. Project dependencies cannot be updated for quite a while and no fixes are coming

**Comment by jlipps** [comment]
we don't pin deps here because renovate keeps them up to date ideally very frequently. it might have been hoped that the same was tree of teen process

**Comment by mykola-mokhnach** [comment]
@jlipps Unfortunately this does not work. Renovate has multiple failed pull requests and they cannot be resolved because of conflicts. The module has been for weeks in such broken state. I would rather keep deps unpinned, but the module itself would be stable.

**Comment by boneskull** [comment]
tbh I'd rather fix what renovate is having issues with instead of unpinning the deps.  pinning them is good, and having multiple versions of dependencies installed is extremely common and likely unavoidable. unless there's some sort of other requirement here, e.g. you need to bundle stuff and keep size small, I feel it's not really harmful enough to unpin. 

as a consumer of teen process, you can always use overrides in yr package.json to force that everything is using a particular babel-runtime. 

otoh I am not sure we need babel-runtime at all anymore given the targeted node versions. should figure that out.

**Comment by boneskull** [comment]
I've merged appium/node-teen_process#102 which will address these problems.  It was just a matter of installing the latest `@appium/eslint-config-appium` and all of the required peer deps at once.

**Comment by boneskull** [comment]
published `teen_process` v2.0.2
